src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,34 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalTime;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = "/info")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/index")
```
better to have consistent naming in Controller class and url mapping